pkg/redis/client.go

```diff
@@ -18,6 +18,10 @@ type ErrCmder interface {
 	Err() error
 }
 
+type Pipeliner interface {
```

Comment:
If you generate a mock for this interface, please also add the go:generate comment to it

Reply:
done